docs/authentication/social-connections/facebook.mdx

```diff
@@ -35,10 +35,10 @@ Once you have a OAuth client ID created, click on the newly created ID under **O
 
 ![Retrieving the App ID and App Secret](/docs/images/authentication-providers/facebook/33456604106d3478e0fd6b583bdde7c4f5563641-1467x969.png)
 
-In the Clerk Dashboard, go to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)** and enable Facebook. In the modal that opened, ensure **Use custom credentials** is enabled and paste the values you obtained during the previous step.
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **Facebook**. Toggle on **Use custom credentials** and paste the values you obtained during the previous step.
 
 ![Adding the valid OAuth Redirect URI](/docs/images/authentication-providers/facebook/c0be80ce8fed2b4d14d04fcbf56318b80295093d-1467x989.png)
 
 Before you close the modal, copy the **Authorized redirect URI**. Go back to the Facebook dashboard, open the **Facebook Login** menu (sidebar) and click **Settings**. Paste the URI you copied before into the **Valid OAuth Redirect URIs** field. Hit **Save Changes**.
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with Facebook is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with Facebook is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with Facebook is now configured for your instance.
```